src/Duende.Bff.DistributedCache/UserSessionCacheExtensions.cs

```diff
@@ -0,0 +1,68 @@
+// // Copyright (c) Duende Software. All rights reserved.
+// // See LICENSE in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Duende.Bff.EntityFramework
+{
+    internal static class UserSessionCacheExtensions
+    {
+        public static async Task StoreUserSessionAsync(
+            this IDistributedCache distributedCache,
+            UserSession session,
+            string applicationDiscriminator,
+            CancellationToken cancellationToken)
+        {
+            Identifier identifier = session.GetKeyIdentifier(applicationDiscriminator);
+            Identifier subjectId = session.GetSubjectIdentifier(applicationDiscriminator);
+            Identifier sessionId = session.GetSessionIdentifier(applicationDiscriminator);
+
+            string? serializedSession = session.ToJson();
+
+            DistributedCacheEntryOptions cacheOptions = new DistributedCacheEntryOptions
+            {
+                AbsoluteExpiration = session.Expires ?? DateTimeOffset.Now.AddDays(1)
```

Comment:
Not sure if this makes sense
The default expiry of the cache entry are the expiry dates of the session.
If nothing is set it is by default one day